tools/jhls/Makefile.sub

```diff
@@ -9,9 +9,13 @@ jhls_LIBS = \
 	libtooling \
 	libhls \
 	libllvm \
+	libmlir \
 	librvsdg \
 	libutil \
 
-jhls_EXTRA_LDFLAGS = $(shell $(LLVMCONFIG) --libs core irReader) $(shell $(LLVMCONFIG) --ldflags) \
+jhls_EXTRA_LDFLAGS = \
+	$(shell $(LLVMCONFIG) --libs core irReader) \
+	$(shell $(LLVMCONFIG) --ldflags) \
```

Comment:
Could we shrink this into one command similarly to how it is in jlm/tooling/Makefile.sub

Reply:
All flags given to a single LLVMCONFIG